packages/backend/src/peripherals/database/ReportRepository.ts

```diff
@@ -31,7 +31,7 @@ export class ReportRepository {
           .andOn('asset_balances.holder_address', '=', 'reports.bridge_address')
       })
 
-    return rows.map((r) => toRecordWithBalance(r, r.balance))
+    return rows.map((r) => toRecordWithBalance(r, '0'))
```

Comment:
revert.

1. getDaily should use isDaily
2. addOrUpdate should recalculate isDaily